universal_notifications/backends/emails.py

```diff
@@ -31,7 +32,7 @@ def send_email(template, to, subject, variables={}, fail_silently=False, cms=Fal
     replace_variables['protocol'] = protocol
     domain = variables['site'].domain
     replace_variables['domain'] = domain
-    for key, value in replace_variables.iteritems():
+    for key, value in (replace_variables.iteritems() if six.PY2 else replace_variables.items()):
```

Comment:
Well, probably less than three years since Django 1.11 will be probably the last to support 2.7, I just prefer not to add special cases for compatibility when there is no such necessity - to avoid cleaning it up later.